### PR TITLE
Add maintainer info for MO and OK

### PR DESCRIPTION
**Pull request by paulboal**
I've added the maintainer and source information for OK and MO.  Excited to be able to contribute.

### CODE REVIEW

**Comment by lazd** [comment]
Awesome! Since you're on more than one scraper, you can add your name in `lib/maintainers` and import it like I did for WY https://github.com/lazd/coronadatascraper/blob/master/lib/maintainers.js

THANK YOU <3

File: scrapers/USA/MO/index.js [hunk 1]
```diff
@@ -14,6 +14,19 @@ const scraper = {
...
+    name: 'Missouri Department of Health and Senior Services'
+  },
+  maintainers: [
+    {
+      name: 'Paul Boal',
+      email: 'paul.boal@amitechsolutions.com',
+      url: 'https://amitechsolutions.com',
+      github: 'paulboal',
+      country: 'USA',
+      flag:
```
Comment (lazd) [inline scrapers/USA/MO/index.js]:
```suggestion
      flag: '🇺🇸'
```

File: scrapers/USA/OK/index.js [hunk 2]
```diff
@@ -14,6 +14,19 @@ const scraper = {
...
+  maintainers: [
+    {
+      name: 'Paul Boal',
+      email: 'paul.boal@amitechsolutions.com',
+      url: 'https://amitechsolutions.com',
+      github: 'paulboal',
+      country: 'USA',
+      flag:
+    }
+  ]
```
Comment (lazd) [inline scrapers/USA/OK/index.js]:
```suggestion
  ],
```

File: scrapers/USA/MO/index.js [hunk 3]
```diff
@@ -14,6 +14,19 @@ const scraper = {
...
+    }
+  ]
```
Comment (lazd) [inline scrapers/USA/MO/index.js]:
```suggestion
  ],
```

**Comment by paulboal** [comment]
Looks like MO already broke my scraper!  I'm going to close this pull request, fix MO, and include these changes in that PR.

**Pull request closed without merging**